RxCocoa/iOS/UITableView+Rx.swift

```diff
@@ -168,15 +168,6 @@ extension Reactive where Base: UITableView {
 extension UITableView {
  
     /**
-    Factory method that enables subclasses to implement their own `delegate`.
-    
-    - returns: Instance of delegate proxy that wraps `delegate`.
-    */
-    public override func createRxDelegateProxy() -> RxScrollViewDelegateProxy {
```

Comment:
Oh crap :(

I mean, your changes are a bandaid, so they are great first step, but hoping we can find some more extensible solution :(